docs/content/docs/getting-started/install-porter.md

```diff
@@ -32,9 +32,7 @@ curl -L https://cdn.porter.sh/$VERSION/install-mac.sh | bash
 ```
 
 **Linux**
-
-```bash
-export VERSION="v1.1.0"
+```bash export VERSION="v1.1.0"
```

Comment:
`export VERSION="v1.1.0" should be on newline, otherwise it won't show up

Reply:
Good catch! This should be resolved now.